docs/concepts/hybrid-approach.md

```diff
@@ -76,6 +81,28 @@ Each entry contains:
 The properties `icm` and `pwaBuild` can use [named capture groups](<https://2ality.com/2017/05/regexp-named-capture-groups.html#replace()-and-named-capture-groups>) and are only used in the _node.js_ process running on the server.
 However, `pwa` and `icmBuild` are used in the client application where [named capture groups are not yet supported by all browsers](https://github.com/tc39/proposal-regexp-named-groups#implementations).
 
+## PWA Adaptions
+
+With version 0.23.0 the PWA was changed to no longer reuse the Responsive Starter Store application types but rather be based upon the newly introduced headless application type for REST Clients - `intershop.REST`.
+This application type would be completely independent of the Responsive Starter Store and is not suited for storefront setups with the Hybrid approach.
+For this reason the PWA needs to be adapted to work together with the Responsive Starter Store again.
+
+**Migration Steps to prepare the PWA for the Hybrid Approach with the Responsive Starter Store**
+
+- use a current PWA version
+- revert the following Git commits from Jun 23, 2020
```

Comment:
I'd rather like to try and implement a configurator like https://github.com/intershop/intershop-pwa/tree/develop/schematics/customization/service-worker for the switching between CMS models  rather than telling people to revert commits...

Reply:
I am not certain if this is a scenario that is needed often enough to implement such a configurator?
I will have to think about that. This is just a simple solution for the few cases where it might be needed.

Reply:
We will start with this revert commits approach and think again about a configurator script if there is need and demand for it.